build.build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
This:

-  packages the application 
- runs twines SA on the package
- installs the package locally
- runs mypy, a SA for python back by guido, on all .py files
- runs shellcheck, an SA for .sh files, on all .sh files
- runs pytests